app/v1/decorators.py

```diff
@@ -0,0 +1,37 @@
+'''Decorators to implement authorization.'''
```

Comment:
Question. Do you think to have `decorators.py` file inside the `v1` folder will be the best way considering that you'll have another `v2` folder for your next challenge?

Reply:
I think its best i have it in the app folder.Am moving decorators.py to the app folder.